index.html

```diff
@@ -145,15 +166,19 @@ <h3>Profesional Art Printing Data</h3>
             </div>
         </section>
                <!-- about me section -->
-        <section class="abt-sct section-container">
+        <section class="abt-sct section-container" id="about">
             <div class="about-section-container">
                 <div class="about-me">
-                    <h1>About Me</h1>
+                    <div class="left">
+                        <h1>About Me</h1>
```

Comment:
- [ ] **[Optional:]**  Kindly change this h1 to a other `hx(h2,h3 etc)` tag. It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browsers